.circleci/config.yml

```diff
@@ -408,6 +408,16 @@ workflows:
           extras: '[unittest,typecheck]'
           python_version: "3.8"
           pandas_version: ">=1"
+          requires:
+            - redshift-s3-itest
+          filters:
+            tags:
+              only: /v\d+\.\d+\.\d+(-[\w]+)?/
+      - test:
+          name: test-3.9
+          extras: '[unittest,typecheck]'
+          python_version: "3.9"
+          pandas_version: ">=1"
```

Comment:
Looks like this one doesn't require `redshift-s3-itest` -- is that intentional for a single, recent python?

Reply:
Yep, that's as expected.  The build starts out with a single unit test task, then a single Redshift integration test, then branches out into everything else.  This PR introduces test-3.9 and relegates test-3.8 to the back of the pack:

![image](https://user-images.githubusercontent.com/14927757/99277702-1c999080-27fc-11eb-8174-7cfc10ad4653.png)